interpreter/package.go

```diff
@@ -60,23 +73,32 @@ func (p *Package) Type() semantic.MonoType {
 }
 func (p *Package) Get(name string) (values.Value, bool) {
 	v, ok := p.object.Get(name)
-	if !ok {
-		return p.options.Get(name)
+	if ok && values.IsOption(v) {
+		// If this value is an option, the option may have
+		// been overriden. Check the override map.
+		if ov, ok := p.options[name]; ok {
+			v = ov
+		}
 	}
-	return v, true
+	return v, ok
 }
 func (p *Package) Set(name string, v values.Value) {
-	p.object.Set(name, v)
+	panic(errors.New(codes.Internal, "package members cannot be modified"))
 }
 func (p *Package) SetOption(name string, v values.Value) {
-	p.options.Set(name, v)
+	// TODO(jsternberg): Setting an invalid option on a package wasn't previously
+	// an error so it continues to not be an error. We should probably find a way
+	// to make it so setting an invalid option is an error.
```

Comment:
what is an invalid option?

Reply:
For whatever reason, the old code seemed to allow this:

```
package foo

option prefix = "default_prefix"
```

```
package main

import "foo"

option foo.sprefix = "my_custom_prefix"
```

The option is spelled wrong in the second one. This seemed like a bug to me, but the old interpreter allowed it so I had no plans to change it in this PR.

Reply:
Agreed. Creating new options is allowed, but I agree creating one from outside the package seems odd. We can disallow that in a separate change.